app/components/avo/index/resource_map_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if @resources.present? %>
+  <div
+    data-selected-resources-name="<%= @resource.model_key %>" data-selected-resources="[]">
+    <%= js_map(helpers.resource_location_markers(@resources), **helpers.resource_map_options(@resource, map_type: :index))  %>
```

Comment:
Another thing I'd like to support here is adding additional markers to the map. The initial use would be for my "near this location" filter, as I'd like to display a marker indicating the search location itself. Others might want to include additional markers for other things on the map.